src/datahike/store.cljc

```diff
@@ -107,12 +120,16 @@
 
 (defmethod default-config :file [config]
   (merge
-   {:path (:datahike-store-path env "datahike-db")}
+   {:path (:datahike-store-path env "datahike-db")
+    :scope #?(:clj (.getHostAddress (InetAddress/getLocalHost))
+              :cljs (throw (ex-info "Not supported yet." {})))}
```

Comment:
Fixed. There are other exceptions thrown in this namespace manually though, which is probably because they are `IllegalArgumentException`s, which is not possible to do with `dt/raise`.